src/standalone/Soil/soil_hydrology_parameterizations.jl

```diff
@@ -7,7 +7,108 @@ export volumetric_liquid_fraction,
     impedance_factor,
     viscosity_factor,
     dψdϑ,
-    is_saturated
+    update_albedo!
+
+"""
+    albedo_from_moisture(surface_liquid_fraction::FT, albedo_dry::FT, albedo_wet::FT)
+
+Calculates pointwise albedo for any band as a function of volumetric soil water content given
+the dry and wet albedo values for that band. Implements eq.1 from Braghiere, R. K et al.
+"""
+function albedo_from_moisture(
+    surface_liquid_fraction::FT,
+    albedo_dry::FT,
+    albedo_wet::FT,
+) where {FT}
+    return albedo_dry * (1 - surface_liquid_fraction) +
+           albedo_wet * surface_liquid_fraction
+end
+
+
+"""
+    update_albedo!(bc::AtmosDrivenFluxBC, p, soil_domain, model_parameters)
+
+Calculates and updates PAR and NIR albedo as a function of volumetric soil water content at
+the top of the soil. If the soil layers are very large, the water content of the top layer
+is used in the calclulation
+"""
+function update_albedo!(bc::AtmosDrivenFluxBC, p, soil_domain, model_parameters)
+    (;
+        ν,
+        θ_r,
+        PAR_albedo_dry,
+        NIR_albedo_dry,
+        PAR_albedo_wet,
+        NIR_albedo_wet,
+        alebdo_calc_top_thickness,
+    ) = model_parameters
+    # ∫H_S_e_dz is the integral of effective saturation from (surface-alebdo_calc_top_thickness) to surface
+    ∫H_S_e_dz = p.soil.sfc_θ_l
+    FT = eltype(soil_domain.fields.Δz_top)
+    # checks if there is at least 1 layer centered within the top soil depth
+    if maximum(soil_domain.fields.Δz_top) < FT(0.5) * alebdo_calc_top_thickness
```

Comment:
If you have a spare sfc_scratch floating around, you can remove the if statement and instead do:
```
    @. albedo_thickness = max(FT(0.5) * alebdo_calc_top_thickness, soil_domain.fields.Δz_top) + sqrt(eps(FT))
```

Reply:
I had something similar earlier. @Sbozzolo suggested using an if statement because the case when there is no layer centered within the top boundary can be calculated more easily without taking integrals. Should I use the if statement, or just set the albedo_thickness to adjust the lower boundary when needed, and integrate?

Reply:
as you prefer! That makes sense too. 

If so, in principle I think it should be the minimum, not the maximum, right? Because if the maximum is > albedo_calc_top_thickness, but the minimum is not, the integral over the some columns will be zero.